Gopkg.toml

```diff
@@ -0,0 +1,71 @@
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+
+required = [
+  # Requiring a subpackage that contains Go code pulls in the whole
+  # package.
+
+  # For verify/update scripts
+  "k8s.io/code-generator/cmd/client-gen",
+  # For verify scripts.
+  "github.com/kubernetes/repo-infra/verify/boilerplate/test",
+]
+
+ignored = ["github.com/kubernetes/repo-infra/kazel"]
+
+[[constraint]]
+  branch = "master"
```

Comment:
Is this is the end state and the plan? Or are you planning to pin this and other dependencies to specific versions? Seems like a lot of room for flakes.

Reply:
The end state is still not quite well-defined. IMO, I think having this file constrain to master is reasonable, since the dependencies are checked in and `Gopkg.lock` has them precisely defined. There is an issue (#59) tracking doing a more-detailed strategy for vendoring.

Reply:
Ok, fair enough.